### DISCUSSION TITLE
Proposal: CUE modules Supply Chain Security

### DISCUSSION THREAD

**Discussion by rogpeppe** [cue-lang/cue#2450] [Proposal]
Proposal #2330 presented a updated modules proposal. This discussion is for a sub-proposal of that: details on some aspects of security with respect to modules.

See here for the sub-proposal text in full: https://github.com/cue-lang/proposal/blob/main/designs/modules/2450-supply-chain-security.md

**Comment by myitcv** [comment]
This proposal was superseded by #2939. As we updated in https://github.com/cue-lang/cue/discussions/2939#discussioncomment-9468945, that proposal (and its sub-proposals) have now been accepted, hence we are closing this discussion to represent the decision on the v3 proposal.
